backend/models/Chat.js

```diff
@@ -2,17 +2,20 @@ const mongoose = require("mongoose");
 const { Schema } = mongoose;
 
 const ChatSchema = new Schema({
-  users: {
-    type: [{ type: Schema.Types.ObjectId, ref: "User" }],
-    validate: {
-      validator: function (users) {
-        return users.length > 0;
-      },
-      message: "A chat must have at least one user.",
-    },
-  },
-  message: [{ type: Schema.Types.ObjectId, ref: "Message" }],
-  polls: [{ type: Schema.Types.ObjectId, ref: "Poll" }],
+	name: {
+		type: String,
+		required: true,
+	},
```

Comment:
I don't think we need a name for each chat since they will just be anoynomous anyway